src/assets/i18n/en.json5

```diff
@@ -3552,6 +3562,64 @@
 
   "media-viewer.playlist": "Playlist",
 
+  "reciter.suggestion.loading": "Loading ...",
```

Comment:
The "reciter" prefix should be removed.  These could all begin with either "publication-claim.suggestion.*" or perhaps "notifications.suggestion.*"